lib/storage/ldp/ldp-file-store.js

```diff
@@ -0,0 +1,165 @@
+'use strict'
+
+const path = require('path')
+const { LdpFileResource, LdpFileContainer } = require('./ldp-file-resource')
+
+class LdpFileStore {
+  /**
+   * @param options {object}
+   *
+   * @param options.fs {object} Expects the `fs-extra` API
+   * @param options.mapper {LegacyResourceMapper}
+   *
+   * @param options.suffixAcl {string}
+   * @param options.suffixMeta {string}
+   * @param options.dataBrowserPath {string}
+   * @param options.suppressDataBrowser {boolean}
+   */
+  constructor (options) {
+    this.fs = options.fs
+    this.mapper = options.mapper
+
+    this.suffixAcl = options.suffixAcl
+    this.suffixMeta = options.suffixMeta
+    this.dataBrowserPath = options.dataBrowserPath
+    this.suppressDataBrowser = options.suppressDataBrowser
```

Comment:
Maybe "databrowser" is too specific, rather something like a viewer?
(An if no viewer path is set, viewer is disabled.)

Reply:
Oh, agreed all the way. These are simply the existing flags - we'll need to refactor/rename them separately.